src/lang/fr.yaml

```diff
@@ -0,0 +1,82 @@
+fr:
+    # agent
+    ## test: The next 2 appear when first starting the chat widget.
+    prompt: 'Écrire un message ...'
+    thinking: 'L’agent réfléchit…'
+    ## test: This is shown after invoking 'list locations' and only getting a single entry in the list (2+ entries show text from the dialog).
+    anything_else: ’Puis-je faire quelque chose d’autre pour vous aider?'
```

Comment:
This value is using a different single quote at beginning (and middle) than at end. Please use same single quote character everywhere in file (it's not just this value). Also, you'll need to wrap in double quotes since using single quote in middle (see other review comments below).